eng/performance/benchmark_jobs.yml

```diff
@@ -123,11 +123,20 @@ jobs:
               ${{ channel }}:
                 _Channel: ${{ channel }}
                 _Configs: CompilationMode=Tiered RunKind="${{ parameters.kind }}"
-                _PerfLabArguments: $(PerfLabArguments)
+                _PerfLabArguments: $(PerfLabArguments) # not necessary?
```

Comment:
I am not finding _PerfLabArguments anywhere else so I think the rename got all the updates. We can probably remove this.

Reply:
Removed